### PR TITLE
FE-608: Update list of supported browsers

### PR DESCRIPTION
**Pull request by bkemper**
Run `npx browserslist` to spotcheck the browserslist.

### CODE REVIEW

File: scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap [hunk 1]
```diff
@@ -3,19 +3,19 @@
 exports[`BrowsersSnapshotGenerator returns list of supported browser 1`] = `
 Object {
   "desktop": Object {
-    "chrome": ">66.0.0",
-    "edge": ">16.0.0",
-    "firefox": ">52.0.0",
-    "opera": ">52.0.0",
-    "safari": ">11.0.0",
+    "chrome": ">58.0.0",
```
Comment (rajumsys) [inline scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap]: is it supposed to be 58 or 68?

Comment (ewandennis) [reply scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap]: 68 came out in 2018-07. Thats a little tight. Here's the Chrome usage spread over the last few weeks:
screen shot 2019-02-05 at 14 36 03

File: scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap [hunk 2]
```diff
@@ -3,19 +3,19 @@
...
+    "edge": ">17.0.0",
+    "firefox": ">60.0.0",
```
Comment (ewandennis) [inline scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap]: Here's the Firefox spread, a chunk of folk on v56 that we're about to disown:

screen shot 2019-02-05 at 14 38 16

File: scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap [hunk 3]
```diff
@@ -3,19 +3,19 @@
...
+    "opera": ">56.0.0",
+    "safari": ">11.1.0",
```
Comment (ewandennis) [inline scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap]: A few Safari 11.0 users too:
screen shot 2019-02-05 at 14 40 59

File: scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap [hunk 4]
```diff
@@ -3,19 +3,19 @@
 exports[`BrowsersSnapshotGenerator returns list of supported browser 1`] = `
 Object {
   "desktop": Object {
-    "chrome": ">66.0.0",
-    "edge": ">16.0.0",
-    "firefox": ">52.0.0",
-    "opera": ">52.0.0",
-    "safari": ">11.0.0",
+    "chrome": ">58.0.0",
+    "edge": ">17.0.0",
```
Comment (ewandennis) [inline scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap]: Edge share is small, few very few <17.

File: scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap [hunk 5]
```diff
@@ -3,19 +3,19 @@
...
+    "firefox": ">60.0.0",
+    "opera": ">56.0.0",
```
Comment (ewandennis) [inline scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap]: Opera share is small, mostly >=v57

File: scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap (hunk 2 above)
Comment (bkemper) [reply scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap]: Disown is a strong word.  I'll identify the impact and if we must we can override it.

File: scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap (hunk 1 above)
Comment (rajumsys) [reply scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap]: i meant whether it's a typo; not as a suggestions to use it.

Comment (bkemper) [reply scripts/tests/__snapshots__/browsersSnapshotGen.test.js.snap]: Unfortunately, it is not a typo.

```

sparkpost/2web2ui FE-608$ npx browserslist "> 1%"
and_chr 71
and_uc 11.8
chrome 71
chrome 70
chrome 58
edge 17
firefox 64
firefox 63
ie 11
ios_saf 12.0-12.1
ios_saf 11.3-11.4
op_mini all
safari 12
```

**Comment by bkemper** [comment]
A couple of topics to hash out.

**Not Used By Babel**

The react-app preset [ignores our browserslist](https://github.com/facebook/create-react-app/blob/master/packages/babel-preset-react-app/create.js#L89) and [sets its own targets](https://github.com/facebook/create-react-app/blob/master/packages/babel-preset-react-app/create.js#L84-L86).  All that discussion about not supporting IE and I think we have been minus polyfills.  Should we change our preset?

**Measuring Impact**

Our browserslist is used by autoprefixer, babel, and `scripts/browsersSnapshotGen.js`.  All three provide a way to detect change when changing your browserslist.  Autoprefixer provides `npx autoprefixer --info` to generate a list of at-rules, selectors, and properties needed for the targets.  Babel provides a [debug option](https://babeljs.io/docs/en/babel-preset-env#debug) to generate a list of plugins for the targets.  Our script has a snapshot test.  All three have different consequences.  If a property is missing from autoprefixer, a user might not get a properly styled DOM element, but our app won't break.  If a plugin is missing, a error will be thrown once that line of code is reached and the user will not be able to complete their task.  If our target browsers is too strict, errors will be reported to Sentry, but as warnings and not errors.  Are there any other considerations?

**Maintenance**

I fear maintaining a minimum version for each browser.  The beauty of "> 1%" is to leverage caniuse database, so you can set it and forget it.  An alternative could be to relax our 1% to something really low like .25% which doesn't change our bundle size right now and configure a [performance budget](https://webpack.js.org/configuration/performance/) to control bundle size moving forward.  Which is better?

**Comment by bkemper** [comment]
Closing for now.

**Pull request closed without merging**